net-im/telegram-desktop/files/telegram-desktop-0.9.49-qmake.patch

```diff
@@ -0,0 +1,72 @@
+--- Telegram/Telegram.pro	2016-05-26 15:02:32.961339302 +0300
++++ Telegram/Telegram.pro	2016-05-26 15:14:42.371349021 +0300
+@@ -10,7 +10,6 @@ CONFIG(debug, debug|release) {
+     DESTDIR = ./../Debug
+ }
+ CONFIG(release, debug|release) {
+-    DEFINES += CUSTOM_API_ID
+     OBJECTS_DIR = ./../ReleaseIntermediate
+     MOC_DIR = ./GeneratedFiles/Release
+     RCC_DIR = ./GeneratedFiles
+@@ -328,10 +327,6 @@ HEADERS += \
+   ./SourceFiles/pspecific_mac.h
+ }
+ 
+-SOURCES += \
+-  ./ThirdParty/minizip/zip.c \
+-  ./ThirdParty/minizip/ioapi.c
+-
+ CONFIG += precompile_header
+ 
+ PRECOMPILED_HEADER = ./SourceFiles/stdafx.h
+@@ -358,19 +353,19 @@ CONFIG(debug, debug|release) {
+ 	QMAKE_LFLAGS_DEBUG += -g -rdynamic -static-libstdc++
+ }
+ 
+-INCLUDEPATH += /usr/local/tdesktop/Qt-5.6.0/include/QtGui/5.6.0/QtGui\
+-               /usr/local/tdesktop/Qt-5.6.0/include/QtCore/5.6.0/QtCore\
+-               /usr/local/tdesktop/Qt-5.6.0/include\
+-               /usr/local/include\
+-               /usr/local/include/opus\
++INCLUDEPATH += ../../../qt/include/QtGui/5.6.0/QtGui\
++               ../../../qt/include/QtCore/5.6.0/QtCore\
++               ../../../qt/include\
+                ./SourceFiles\
+                ./GeneratedFiles\
+-               ./ThirdParty/minizip\
+-               ./../../Libraries/breakpad/src
++               /usr/include/opus\
++               /usr/include/minizip\
++               /usr/include/breakpad
+ 
+-INCLUDEPATH += "/usr/include/libappindicator-0.1"
+-INCLUDEPATH += "/usr/include/gtk-2.0"
++INCLUDEPATH += "/usr/include/libappindicator3-0.1"
++INCLUDEPATH += "/usr/include/gtk-3.0"
+ INCLUDEPATH += "/usr/include/glib-2.0"
++INCLUDEPATH += "/usr/lib/glib-2.0/include"
+ INCLUDEPATH += "/usr/lib/x86_64-linux-gnu/glib-2.0/include"
+ INCLUDEPATH += "/usr/lib/i386-linux-gnu/glib-2.0/include"
+ INCLUDEPATH += "/usr/include/cairo"
+@@ -384,12 +379,15 @@ INCLUDEPATH += "/usr/include/dee-1.0"
```

Comment:
True, in my ebuild I simply deleted all hardcoded paths and re-added them with pkg-config - https://github.com/rindeal/gentoo-overlay/blob/414b575b4877c8acc64a4873a5cf69b7efc7b162/net-im/telegram/telegram-0.9.49.ebuild#L108-132